integrated_channels/moodle/models.py

```diff
@@ -73,6 +107,38 @@ class MoodleEnterpriseCustomerConfiguration(EnterpriseCustomerPluginConfiguratio
         )
     )
 
+    decrypted_password = EncryptedCharField(
+        max_length=255,
+        verbose_name="Encrypted Webservice Password",
+        blank=True,
+        help_text=_(
+            "The encrypted API user's password used to obtain new tokens."),
+        default="",
+    )
```

Comment:
[nit] Once we address the above feedback, please make sure to add back null=True and remove default value.

Reply:
Why did we use this filed name decrypted_password except for encrypted_password?

Reply:
I think it's for extra caution so whoever accesses these values in app knows this data will be available in decrypted form. It's only saved in encrypted form in DB at rest.